src/gcovr/utils.py

```diff
@@ -31,16 +31,14 @@
 
 LOGGER = logging.getLogger("gcovr")
 
-MD5_KWARGS = {"usedforsecurity": False} if sys.version_info >= (3, 9) else {}
-
```

Comment:
:+1: excellent for spotting + removing this now-unnecessary check!

Reply:
I always search for `version_info` if dropping support for a specific version.